ios/FluentUI.Demo/FluentUI.Demo/Demos/PillButtonBarDemoController.swift

```diff
@@ -76,18 +76,19 @@ class PillButtonBarDemoController: DemoController {
 
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
-        if let window = view.window {
-            onBrandBar?.backgroundColor = UIColor(light: Colors.primary(for: window), dark: Colors.navigationBarBackground)
-            customBar?.backgroundColor = UIColor(light: Colors.primary(for: window), dark: Colors.navigationBarBackground)
-        }
+        let backgroundColorLight = UIColor(dynamicColor: view.fluentTheme.aliasTokens.colors[.brandBackground1])
+        onBrandBar?.backgroundColor = UIColor(light: backgroundColorLight, dark: UIColor(colorValue: GlobalTokens.neutralColors(.grey16)))
```

Comment:
Maybe you should be caching the dark color as well

Reply:
I think they might be caching the light mode color because DynamicColor's .light property is internal, so if you want to capture it outside of our library you need to do something like this. Dark is using a ColorValue, though, so it's probably fine to not cache it.